Dockerfile

```diff
@@ -33,4 +33,4 @@ EXPOSE 7896
 WORKDIR /$PRJ_DIR/target
 
 # Run application in container
-CMD ["java", "-jar", "lpvs-1.0.2.jar"]
+CMD ["java", "-jar", "lpvs-*.jar"]
```

Comment:
@tdrozdovsky My lpvs container can't find the `lpvs-*.jar` file. Does your container work accordingly?

Reply:
Please see #166